python/ray/serve/scripts.py

```diff
@@ -522,6 +522,7 @@ def run(
                     serve.run(
                         app, name=name, route_prefix=route_prefix, host=host, port=port
                     )
+                    cli_logger.success("Redeployed app successfully.")
```

Comment:
Non-blocker: Should we also add the name of the app so we know which one is redeployed? 🙃 Not a blocker and not part of the ask in the issue, just wondering.